tools/devops/automation/templates/build/build-stage.yml

```
@@ -145,3 +153,23 @@ jobs:
     parameters:
       enableDotnet: ${{ parameters.enableDotnet }}
       sbomFilter: '*.nupkg;*.pkg;*.msi'
+
+# Job that runs on a vm that downlods the artefacts information and adds a gihub comment pointing to the results of the build.
+- job: artefacts_github_comment
```

```suggestion
- job: artifacts_github_comment
```
This one may affect other places